JavaAssignment/src/swapnali/Q1_LargestThreeNumbers.java

```diff
@@ -0,0 +1,32 @@
+package swapnali;
+import java.util.Scanner;
+
+public class Q1_LargestThreeNumbers {
+	public static void main(String[] args) {
+		Q1_LargestThreeNumbers largestThreeNumbers = new Q1_LargestThreeNumbers();
+		largestThreeNumbers.printLargestOfThreeNumbers();
+	}
+
+	void printLargestOfThreeNumbers()
+	{
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Enter the numbers: ");
+		int num1 = scanner.nextInt();
+		int num2 = scanner.nextInt();
+		int num3 = scanner.nextInt();
+		scanner.close();
+
+		if (num1 > num2 && num1 > num3) {
+			System.out.println("First number is largest.");
+		}
+		else if (num2 > num1 && num2 > num3) {
```

Comment:
Here swapnali you already compared num1 n num2 know 19th line. If num1 is greater than. Then it prints num1 large then why we again comparing num1 n num2 . Think on this